PostHog/posthog-ios
# Update device type

**What does this PR do?**
This updates the device type property sent by SDK to conform examples seen in documentation.

**Where should the reviewer start?**
Just check `PHGPostHogIntegration` file.

**How should this be manually tested?**
Trigger any event and check that `$device_type` is set with one of the implemented values.

**Any background context you want to provide?**
Old value "ios" doesn't represent the a device type. Even less for tvOS apps.
